_components/form-templates.md

```diff
@@ -352,8 +307,8 @@ lead: Patterns for some of the most commonly used forms on government websites.
     </ul>
   </div>
 </div>
-
-<div class="preview" id="ssn-form">
+<h2 class="usa-heading" id="ssn-form">Social security number form</h2>
```

Comment:
Well, generally we should be using sentence case as per the 18F content style guide (https://pages.18f.gov/content-guide/capitalization/) but your question made me stop to think about SSN being a proper noun. So far the writing lab seems to think Social Security Number should be capitalized.